genie-demo/src/main/docker/client/example/run_hadoop_job.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python2.7
+
+# Copyright 2016 Netflix, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##################################################################################
+# This script assumes setup.py has already been run to configure Genie and that
+# this script is executed on the host where Genie is running. If it's executed on
+# another host change the localhost line below.
+##################################################################################
+
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
It looks like you don't need any of these imports. Same in the other `run_*_job.py` files.